src/plugins/intel_cpu/src/ngraph_transformations/convert_fq_rnn_to_quantized_rnn.cpp

```diff
@@ -0,0 +1,221 @@
+                lstm_seq->get_activations(),
+                lstm_seq->get_clip());
+
+            rnn_quantized_tr->set_overridden_output_type(hidden_state.get_element_type(), 1);
+            rnn_quantized = rnn_quantized_tr;
+        } else if (const auto gru_seq = ngraph::as_type_ptr<ngraph::opset9::GRUSequence>(rnn)) {
+            const auto& sequence_length = pattern_map.at(sequence_length_m);
+
+            auto rnn_quantized_tr = std::make_shared<ngraph::op::TypeRelaxed<ngraph::opset9::GRUSequence>>(
+                std::vector<ngraph::element::Type>{ element::f32, element::f32, element::f32, element::f32, element::f32, element::f32},
+                std::vector<ngraph::element::Type>{ element::f32, element::f32 },
+                ngraph::op::TemporaryReplaceOutputType(activation, element::f32).get(),
+                ngraph::op::TemporaryReplaceOutputType(hidden_state, element::f32).get(),
+                ngraph::op::TemporaryReplaceOutputType(sequence_length, element::f32).get(),
+                ngraph::op::TemporaryReplaceOutputType(weights, element::f32).get(),
+                ngraph::op::TemporaryReplaceOutputType(r_weights, element::f32).get(),
+                ngraph::op::TemporaryReplaceOutputType(bias, element::f32).get(),
+                gru_seq->get_hidden_size(),
+                gru_seq->get_direction(),
+                gru_seq->get_activations(),
+                gru_seq->get_activations_alpha(),
+                gru_seq->get_activations_beta(),
+                gru_seq->get_clip(),
+                gru_seq->get_linear_before_reset());
+
+            rnn_quantized_tr->set_overridden_output_type(hidden_state.get_element_type(), 1);
+            rnn_quantized = rnn_quantized_tr;
+        } else {
+            return false;
+        }
+
+        // input scales (Multiply per tensor) and weights_scales (Multiply per multiple dimensions) must be present
+        const auto& input_scale_output   = pattern_map.at(input_scale_X);
+        const auto& weights_scale_output = pattern_map.at(weights_scale_W);
+        // extract constant values
+        const auto input_scale_constant   = std::dynamic_pointer_cast<ngraph::opset9::Constant>(input_scale_output.get_node_shared_ptr());
+        const auto weights_scale_constant = std::dynamic_pointer_cast<ngraph::opset9::Constant>(weights_scale_output.get_node_shared_ptr());
+
+        if (!input_scale_constant || !weights_scale_constant)
+            return false;
+
+        const float* input_scale_ptr = input_scale_constant->get_data_ptr<float>();
+        if (*input_scale_ptr == 0.f)
+            throw ngraph::ngraph_error("Cannot handle zero input scale");
+
+        const float                 input_scale  = 1 / *input_scale_ptr;
+        const std::vector<float> weights_scales  = weights_scale_constant->get_vector<float>();
+
+        auto& runtime_info = rnn_quantized->get_rt_info();
+
+        runtime_info["inputScale"]    = input_scale;
```

Comment:
I believe we need generic comment for this transformation that will describe why we use rt_info for quantization information propagation instead of introducing new custom op.